### PR TITLE
Support non texture.sampler specified glTF file

### LINKED ISSUE #11260: Latest spec glTF 2.0 Model are not displayed properly
##### Description of the problem 

I am testing glTF 2.0 sample models and Three.js glTF 2.0 Loader.
https://github.com/cx20/gltf-test/tree/2.0#gltf-20-pbr-models

However, it seems that some PBR models are not displayed properly.
https://cdn.rawgit.com/cx20/gltf-test/b6900f6f4d755a71691aaef5a19279da7472b69f/examples/threejs/index.html?category=tutorialModels&model=Corset&scale=1&type=glTF

image

I think that it was caused by updating to the latest specification about 10 days ago.
https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Corset

##### Three.js version

- [x] Dev
- [x] r85
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [x] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

##### Hardware Requirements (graphics card, VR Device, ...)

ThinkPad X260 + Windows 10 + Intel HD Graphics 520

### PR DESCRIPTION
**Pull request by takahirox**
This PR fixes #11260 by accepting non texture.sampler specified glTF file.

### CODE REVIEW

File: examples/js/loaders/GLTF2Loader.js [hunk 1]
```diff
@@ -1430,16 +1430,12 @@ THREE.GLTF2Loader = ( function () {
 
 							_texture.type = texture.type !== undefined ? WEBGL_TEXTURE_DATATYPES[ texture.type ] : THREE.UnsignedByteType;
 
-							if ( texture.sampler !== undefined ) {
+							var sampler = texture.sampler === undefined ? {} : json.samplers[ texture.sampler ];
```
Comment (mrdoob) [inline examples/js/loaders/GLTF2Loader.js]: How about...

```js
var sampler = json.samplers[ texture.sampler ];
if ( sampler === undefined ) sampler = {};
```

Comment (takahirox) [inline examples/js/loaders/GLTF2Loader.js]: `json.samplers` is optional.
So, how about this?

    var samplers = json.samplers || {};
    var sampler = samplers[ texture.sampler ] || {};

Comment (mrdoob) [inline examples/js/loaders/GLTF2Loader.js]: Looks good 👍

**Comment by takahirox** [comment]
Updated!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**